functions/model/newsroom/lib.js

```diff
@@ -32,4 +32,19 @@ exports.getAllNews = function() {
     });
 
     return Promise.resolve(promise);
+};
+
+exports.editNewsById = function(uid, imageUrl, name, link) {
+    const query = db.collection("News").doc(uid).update({
+        ImageUrl: imageUrl,
+        Name: name,
+        Link: link,
+});
+    return Promise.resolve(query);
+};
+
+exports.deleteNewsById = function(uid) {
+    const query = db.collection("News").doc(uid).delete();
+
+    return Promise.resolve(query);
```

Comment:
Should change the status to deleted and upon get only get the data with status of OK